core/src/test/java/com/linecorp/armeria/internal/annotation/AnnotatedHttpServiceFactoryTest.java

```diff
@@ -122,6 +131,35 @@ public void ofUserDefinedRepeatableDecorator() throws NoSuchMethodException {
         assertThat(udd2.value()).isEqualTo(2);
     }
 
+    @org.junit.jupiter.api.Test
```

Comment:
I wouldn't recommend mixing JUnit 4 and 5 annotations. If you are not going to migrate this test case to JUnit 5, I recommend sticking to JUnit 4.

Reply:
Sure. Will use junit4 annotations.